apps/next-evans/pages/friend/[friend_name].tsx

```diff
@@ -0,0 +1,68 @@
+import { PrismaClient } from '@prisma/client';
+import type { GetStaticPaths, GetStaticProps } from 'next';
+import FriendBlock from '@/components/friend/FriendBlock';
+import FeaturedQuoteBlock from '@/components/friend/FeaturedQuoteBlock';
+
+const prisma = new PrismaClient();
```

Comment:
i think i would feel slightly better for you to move this into both of the functions that need it. it's probably fine this way, but i feel a little funny about creating this in the global scope, because it means that just  _importing_ this file might have side-effects, which can cause subtle issues, and there's not any real cost (other than one line being repeated) to moving it into the functions that get exported.